packages/react/src/components/Pagination/experimental/Pagination-story.js

```diff
@@ -74,3 +74,68 @@ export const WithNoSizerChildInputOrChildSelector = () => (
 
 WithNoSizerChildInputOrChildSelector.storyName =
   'with no sizer, child input, or child selector';
+
+export const Playground = (args) => <Pagination {...args} />;
+
+Playground.argTypes = {
+  className: {
+    control: false,
+  },
+  children: {
+    control: false,
+  },
+  id: {
+    control: false,
+  },
+  forwardText: {
+    control: {
+      type: 'text',
+    },
+    defaultValue: 'Next page',
+  },
+  disabled: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: 'false',
+  },
+  itemRangeText: {
```

Comment:
`itemText` also needs `control: false`

Reply:
In both this an the other story as well I think

Reply:
good catch, added!